Makefile

```diff
@@ -48,14 +48,14 @@ ifndef GIT_VERSION
     GIT_VERSION := $(GIT_SHA)-dev
   endif
 endif
-GO_LDFLAGS ?= -X 'main.Version=$(GIT_VERSION)'
+GO_LDFLAGS ?= -X 'github.com/stolostron/policy-cli/internal.version=$(GIT_VERSION)'
```

Comment:
Oh nice, I didn't know this could set an unexported value in the `internal` package!